.github/workflows/staging-build-data-prepper.yml

```diff
@@ -16,23 +16,24 @@ jobs:
     - name: Configure AWS Credentials
       uses: aws-actions/configure-aws-credentials@v1
       with:
-        aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
-        aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        aws-access-key-id: ${{ secrets.AWS_STAGING_ACCESS_KEY_ID }}
+        aws-secret-access-key: ${{ secrets.AWS_STAGING_SECRET_ACCESS_KEY }}
         aws-region: us-east-1
 
+    - name: required utilites
+      run: release-tools/scripts/required_packages.sh
+
     - name: S3 sync artifacts
       run: |
         #!/bin/bash
```

Comment:
We can remove this line, as git actions assign bash by default.

Reply:
Removed